objurl/objurl.go

```diff
@@ -122,7 +122,10 @@ func (o *ObjectURL) Absolute() string {
 
 // Relative returns a URI reference based on the calculated prefix.
 func (o *ObjectURL) Relative() string {
-	return o.relativePath
+	if o.relativePath != "" {
```

Comment:
This change is required since Relative() for non-wildcard urls return empty. For example, `s3://bucket/key` has empty relative path and while using it with --parents flag, it constructs wrong destination URL. 
  
```
s5cmd cp --parents testfile "testdir/*" s3://bucket

# we need both the content inside /testdir and testfile.
```
Above example was not working before since testfile creates a target `s3://bucket` (which is wrong actually).

Reply:
👍